src/richie/apps/search/indexers/courses.py

```diff
@@ -183,9 +192,35 @@ def get_data_for_es(cls, index, action):
                     simple_text.body
                 )
 
+            # Make sure we get title information for categories in the same request
+            categories = (
```

Comment:
this is not retrieving categories but pages related to categories. So maybe?
```suggestion
            category_pages = (
```